fastlane/.env

```diff
@@ -1,4 +1,4 @@
 KEYCHAIN_NAME=Wire-Distribution
 MATCH_KEYCHAIN_NAME=Wire-Distribution
-IOS_VERSION='16.4'
-IPHONE_MODEL='iPhone 8'
\ No newline at end of file
+IOS_VERSION='17.0'
+IPHONE_MODEL='iPhone 15'
```

Comment:
Would be better to stick to most common device: iPhone 11 or with dynamic island iPhone 14, what do you think?

Reply:
I think it'd make sense to go with something with dynamic island as that will probably soon be the majority of phones. But there's only a few pixels difference between iPhone 14 (390 × 844 px) and iPhone 15 (393 × 852 px) so I don't see a problem with picking 15 over 14.

Reply:
there's not a huge difference, but it would be good to stay at iPhone 14 to cover most common usage

Reply:
sure, I'll re-record the snapshots

Reply:
done